vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -184,6 +196,44 @@ func ObserveQualityMetricsRecommendationMissing(usage float64, isOOM bool, resou
 	observeUsageRecommendationDiff(usage, 0, true, isOOM, resource, updateMode)
 }
 
+// ObserveRecommendationChange records relative_recommendation_changes metric.
+func ObserveRecommendationChange(previous, current corev1.ResourceList, updateMode *vpa_types.UpdateMode, vpaSize int) {
+	// This will happen if there is no previous recommendation, we don't want to emit anything then.
+	if previous == nil {
+		return
+	}
+	// This is not really expected thus a warning.
+	if current == nil {
+		klog.Warningf("Cannot compare with current recommendation being nil. VPA mode: %v, size: %v", updateMode, vpaSize)
+		return
+	}
+
+	for resource, amount := range current {
+		newValue := quantityAsFloat(resource, amount)
+		oldValue := quantityAsFloat(resource, previous[resource])
+
+		if oldValue > 0.0 {
+			diff := newValue/oldValue - 1.0 // -1.0 to fall into buckets from -1.0 to 100.0
```

Comment:
-1.0 to report decrease as negative value.

Reply:
(meaning can we clarify the intent more in the comment, fitting into buckets is just one of the reasons)

Reply:
Done.